tsconfig.json

```diff
@@ -21,9 +21,10 @@
       "@sections/*": ["src/sections/*"],
       "@constants/*": ["src/constants/*"],
       "@services/*": ["src/services/*"],
-      "@hooks/*": ["src/hooks/*"]
+      "@hooks/*": ["src/hooks/*"],
+      "@formatters/*": ["src/formatters/*"],
     }
   },
-  "include": ["src/*"],
+  "include": ["src", "src/*"],
```

Comment:
do you need both entries?

Reply:
Right, I just updated to `"include": ["src/**/*"]`, custom paths like `@components/Button` were not being resolved correctly